packages/components/src/interfaces/internal/Navbar.ts

```diff
@@ -27,9 +32,16 @@ export interface BaseNavbarProps {
 export interface NavbarProps extends BaseNavbarProps {
   logoUrl: string
   logoAlt: string
+  callToAction?: NavbarCallToAction & {
+    url: string
+  }
   site: IsomerSiteProps
 }
 
 export interface NavbarClientProps extends BaseNavbarProps {
+  callToAction?: NavbarCallToAction & {
+    referenceLinkHref?: string
+    isExternal: boolean
```

Comment:
[nit] `isExternal` is a computed property of `referenceLinkHref`, so more in favour of just computing it at compile time rather than storing it as data

Reply:
oh `NavbarClientProps` are all computed at build time